xkit.css

```diff
@@ -982,7 +982,5 @@
 }
 
 .post_brick .post_controls .post_controls_inner {
-    width: 100%;
-    white-space: normal;
-    height: auto;
+	white-space: nowrap;
```

Comment:
Oh, fun! It does match the *majority* of the file, but not its immediate neighbors. I hadn't noticed because I only checked whitespace using a regex in js files. Maybe I'll PR consistent CSS indentation separately, but I'll make this match for now.